index.js

```diff
@@ -81,6 +82,26 @@ Y18N.prototype._readLocaleFile = function () {
   this.cache[this.locale] = localeLookup
 }
 
+Y18N.prototype._resolveLocaleFile = function (directory, locale) {
+  var file = path.resolve(directory, './', locale + '.json')
+  if (this.fallbackToLanguage && !this._fileExistsSync(file) && ~locale.lastIndexOf('_')) {
+    // attempt fallback to language only
+    var languageFile = path.resolve(directory, './', locale.split('_')[0] + '.json')
+    if (this._fileExistsSync(languageFile)) file = languageFile
+  }
+  return file
+}
+
+// this only exists because fs.existsSync() "will be deprecated"
+// see https://nodejs.org/api/fs.html#fs_fs_existssync_path
+Y18N.prototype._fileExistsSync = function (file) {
+  try {
+    return fs.statSync(file).isFile()
+  } catch (err) {
+    return false
```

Comment:
Ignoring an error stinks, but at least we're not using a deprecated function -- I think your right to avoid throwing in this library, we always want it to simply fallback to returning a default string.